independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Jira Connector",
```

Comment:
@Flow-Joe Connector title should not contain a word called "Connector"

Documentation - [Adding title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)